commands/ssh.go

```diff
@@ -50,6 +53,7 @@ You may specify the user to login with by passing the `+"`"+`--%s`+"`"+` flag. T
 	AddBoolFlag(cmdSSH, doctl.ArgsSSHAgentForwarding, "", false, "Enable SSH agent forwarding")
 	AddBoolFlag(cmdSSH, doctl.ArgsSSHPrivateIP, "", false, "SSH to Droplet's private IP address")
 	AddStringFlag(cmdSSH, doctl.ArgSSHCommand, "", "", "Command to execute on Droplet")
+	AddBoolFlag(cmdSSH, doctl.ArgCommandRetry, "", false, "Retry for a successful SSH connection with Droplet")
```

Comment:
I wonder if it would make sense to just use the global `--http-retry-max` flag? On the one hand conflating HTTP and SSH retries might be a bit unintuitive. On the other, it's would be respecting the general intention to retry transient failures.

Reply:
Ya I can see that. We could meet somewhere in the middle with `ssh-retry-max`. That makes it also consistent with the rest of the `ssh` flags i.e. `ssh-keys`,  `ssh-agent-forwarding` , `ssh-private-ip` etc.